web/content/js/versions.js

```diff
@@ -203,6 +215,14 @@ const navVersions = [
             "title": "Quickstart Guide",
             "path": "Quickstart"
           },
+          {
+            "title": "Updating Katello",
```

Comment:
I'd say installing is more important than upgrading/updating, so the previous position was correct.

Reply:
My point was: it does not make sense to delete "upgrading" in line 126 and then add updating + upgrading in line 218+.

Reply:
Addressed. I hope I understood correctly.